tools/ndt/main.cpp

```diff
@@ -3,15 +3,17 @@
 
 // nsc input/simple_shader.hlsl -T ps_6_0 -E Main -Fo output/shader.ps
 
+#include "nbl/system/IApplicationFramework.h"
+
 #include <iostream>
 #include <cstdlib>
-#include <vector>
+
 
 
 bool no_nbl_builtins;
 
 
-bool noNblBuiltinsEnabled(const std::vector<std::string>& args)
+bool noNblBuiltinsEnabled(const core::vector<std::string>& args)
```

Comment:
make it a method and you don't need to pass the `argv` around since its a member

Reply:
OK. I think I'll just remove it and write the loop directly.

Reply:
anyway argument parsing is @Hazardu's job, you can hardcode all inputs and stuff meant to be gathered from cmdline